notebooks/cleaning_data.ipynb

```diff
@@ -0,0 +1,2892 @@
+{
```

Comment:
<p>This summary is misleading since, with the exception of the "labeled" feature type, we handled all types of features which include direct and indirect types of missing data.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mawada-sweis/Clustering-Analysis/pull/21/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='45'/>